lib/src/lib/table/components/table-cell/table-cell.component.ts

```diff
@@ -89,4 +92,17 @@ export class Lab900TableCellComponent<T = any> {
   public getCellHeaderSvgIcon(): string {
     return readPropValue<TableCell<T>>(this.cell.cellHeaderSvgIcon, this.cell);
   }
+
+  public getCellFooter(): string {
+    if (this.cell.footer) {
+      if (typeof this.cell.footer === 'function') {
+        return this.cell.footer(this.data, this.cell);
+      }
+      return this.cell.footer;
+    }
+  }
+
+  public getCellFooterClass(): string {
```

Comment:
I don't think this function is needed? you can put it in the HTML directly

Reply:
done